tests/basics/builtin_range.py

```diff
@@ -54,6 +54,32 @@
 print(range(1, 100, -5)[5:15:-3])
 print(range(1, 100, -5)[15:5:3])
 
+for i in range(24, 67):
```

Comment:
It feels like this new set of tests should be in a separate file, because:
- I don't think it'll work on ports without big int (maybe I'm wrong there??), so should be suffixed with `_intbig.py`.
- Since it doesn't really work on CPython, it should probably have a .py.exp file.

Also, maybe a short comment at the top describing what the test is testing.

Reply:
I updated the test so that it does not depend on long int support or a specific sys.maxsize value. I also added a cpydiff for some additional oddities I discovered while testing.